### PR TITLE
HIVE-24672: compute_stats_long.q fails for wrong reasons

### PR DESCRIPTION
**Pull request by mustafaiman**
Change-Id: I61d941dcbf86fb2dd45772fc658b3dc887325bd0

### CODE REVIEW

**Review by zeroflag — APPROVED** [review]
LGTM

File: ql/src/test/queries/clientnegative/compute_stats_long.q [hunk 1]
```diff
@@ -3,5 +3,5 @@ create table tab_int(a int);
 -- insert some data
 LOAD DATA LOCAL INPATH "../../data/files/int.txt" INTO TABLE tab_int;
 
--- compute stats should raise an error since the number of bit vectors > 1024
-select compute_stats(a, 10000) from tab_int;
+-- this should raise an error since the number of bit vectors has a hard limit at 1024
+select compute_stats(a, 'fm', 10000) from tab_int;
```
Comment (abstractdog) [inline ql/src/test/queries/clientnegative/compute_stats_long.q:7]: does this limit apply only to fm algo?

Comment (mustafaiman) [reply ql/src/test/queries/clientnegative/compute_stats_long.q:7]: yes see https://github.com/apache/hive/blob/5d735c20ffb6a8624382155a3e90e650db2fb80e/standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/common/ndv/NumDistinctValueEstimatorFactory.java#L64

Comment (abstractdog) [reply ql/src/test/queries/clientnegative/compute_stats_long.q:7]: thanks

Comment (abstractdog) [reply ql/src/test/queries/clientnegative/compute_stats_long.q:7]: +1

**Review by abstractdog — APPROVED** [review]

**Pull request closed without merging**